lib/client/layout.js

```diff
@@ -114,10 +132,10 @@ BlazeLayout._render = function _render(layout, regions) {
     currentData = BlazeLayout._regionsToData(regions);
 
     currentLayout = Blaze._TemplateWith(currentData, function() {
-      return Spacebars.include(Template[layout]);
+      return Spacebars.include(BlazeLayout._getTemplate(layout, rootDomNode));
     });
 
-    Blaze.render(currentLayout, rootDomNode);
```

Comment:
If the DOM root is rendered as a template (or in our case a component), then it is good that the layout template (or component) has parent view properly set because then you can traverse the template (component) tree nicely. This means that all layout regions have the same root (root template), instead of them being seen as separate templates.

If `rootDomNode` was not rendered using a template (or component), `Blaze.getView` should return `null` so the behavior would be as before.